MarsRover/Controller.cs

```diff
@@ -19,60 +19,25 @@ public Controller(IInput input, IOutput output, IMapInput mapInput)
             _mapInput = mapInput;
         }
 
-        /*
-            Setup:
-                1/ ask for fileinput
-                2/ does file exist?
-                if file exists:
-                    validate the file
-                    if not
-                        keep asking if file is not valid
-                    if valid
-                        parse the map
-                        DisplayMap(map)
-                ask for rover direction from user
-                    validate direction
-                    if not valid
-                        keep asking for valid direction
-                    if valid
-                    ask for rover location from user
-                        validate location
-                        if not valid
-                            keep asking for valid location
-                        if valid
-                            is there an obstacle?
-                            yes - repeat asking for location input
-                            no - create rover
-                            DisplayMap(map, rover)
-                ** test to return map with rover when setup complete
-        */
-
-        // Move logic:
-        // 1/ GetTargetLocation to determine where Rover should Move to
-        // 2a/ use Map.GetSquareAtLocation in order to find Square -  ?? HasObstacle - Controller
-        // 2b/ use Map.HasObstacle on the square to check if there is an obstacle - Controller
-        // 3/ if no obstacle, set Rover.Location to the location of that square - ?? Move
-        // if there is an obstacle, Rover.Location remains the same
-
         public void Setup()
         {
             _output.WriteLine(Messages.Title);
-            Map = InitialiseMap();
+            InitialiseMap();
             _output.WriteLine(OutputFormatter.DisplayMap(Map));
-            Rover = InitialiseRover();
+            InitialiseRover();
             _output.WriteLine(OutputFormatter.DisplayMap(Map, Rover));
         }
 
-        private Map InitialiseMap()
+        private void InitialiseMap()
         {
             _output.WriteLine(Messages.RequestMapInput);
-            var filePath = _input.Read(); 
-            var fileExists = _mapInput.FileExists(filePath); // TODO: file doesn't exist
+            var filePath = _input.ReadLine(); 

```

Comment:
can the code related to getting & validating the file path be extracted to its own method?